perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -40,23 +40,23 @@ then
 fi
 
 run_load_test_and_fetch_metrics(){
-  GCSFUSE_FIO_FLAGS=$1
-  UPLOAD_FLAGS=$2
-  BUCKET_NAME=$3
-  SPREADSHEET_ID=$4
-  GSHEET_CREDENTIALS=$5
-  TEST_TYPE=$6
-  GCSFUSE_LS_FLAGS=$7
-  LS_CONFIG_FILE=$8
+  gcsfuse_fio_flags=$1
+  upload_flags=$2
+  bucket_name=$3
+  spreadsheet_id=$4
```

Comment:
Do we really needs these arguments which are not changing for any cases like "upload_flags" or "gcsfuse_flags". can't we access it without passing it to the function as it is global.

Reply:
Updated with common global flags.